akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -117,39 +117,39 @@ specification, which Akka is a founding member of.
 The user of the library does not have to write any explicit back-pressure handling code — it is built in
 and dealt with automatically by all of the provided Akka Streams processing stages. It is possible however to add
 explicit buffer stages with overflow strategies that can influence the behaviour of the stream. This is especially important
-in complex processing graphs which may even sometimes even contain loops (which *must* be treated with very special
+in complex processing graphs which sometimes even may contain loops (which *must* be treated with very special
```

Comment:
Using thre three words above, I would say: "... which may sometimes even contain loops...". 
But, I probably wouldnt use those three words like that, but instead say something like "... which may sometimes contain loops ..." or "... may even contain loops ...". That just sounds like it flows more naturally to me.